File: BlackJack.DataAccess/Entities/Round.cs
﻿using BlackJack.Shared.Enums;
using System.Collections.Generic;

namespace BlackJack.DataAccess.Entities
{
    public class Round : BaseEntity
    {
        public long GameId { get; set; }
        public virtual Game Game { get; set; }

        public long PlayerId { get; set; }
        public virtual Player Player { get; set; }

        public RoundState State { get; set; }

        public virtual ICollection<RoundCard> Cards { get; set; }
    }
}

File: BlackJack.TryAngular/ClientApp/src/app/_models/round.ts
import { PlayerType } from '../_enums/player.type';
import { Card } from './card';
import { RoundState } from '../_enums/round.state';

export class Round {
    playerName?: string;
    playerType?: PlayerType;
    cards?: Card[];
    state?: RoundState;
    score?: number;
    stringifiedCards?: string;
}